ImmersionThemePlugin.inc.php

```diff
@@ -460,4 +494,26 @@ public function hasEmbeddedCSS($hookName, $args) {
 			'themePath' => $request->getBaseUrl() . "/" . $this->getPluginPath(),
 		));
 	}
+
+	/**
+	 * Get all defined sectin colors indexed by 'issueId_sectionId'
+	 */
+	public function getAllSectionColors() {
+		$issueDao = DAORegistry::getDAO('IssueDAO');
+		$issues = $issueDao->getPublishedIssues(Application::get()->getRequest()->getContext()->getId());
```

Comment:
Can we avoid this call? I'm worrying that it introduces additional database call to retrieve all issue objects per each request.